conda_build/cli/main_render.py

```diff
@@ -196,11 +197,18 @@ def execute(args, print_results=True):
                                  no_download_source=args.no_source,
                                  variants=args.variants)
 
+    if args.file and len(metadata_tuples) > 1:
+        log.warning("Multiple variants rendered. "
+                    "Only one will be written to the file you specified ({}).".format(args.file))
+
     if print_results:
         if args.output:
             with LoggingContext(logging.CRITICAL + 1):
                 paths = api.get_output_file_paths(metadata_tuples, config=config)
                 print('\n'.join(sorted(paths)))
+            if args.file:
+                (m, _, _) = metadata_tuples[-1]
```

Comment:
```suggestion
                m = metadata_tuples[-1][0]
```

Reply:
When I first wrote this code, I couldn't make up my mind between these two choices.  I thought it might be nice to use same syntax that the for-loop below uses.  But OK.  I've changed it now.